flows.yaml

```
@@ -1,4 +1,39 @@
 integrations:
+    aircall-basic:
+        actions:
+            create-user:
+                description: Creates a user in Aircall.
+                output: User
+                endpoint: POST /users
+                input: CreateUser
+            delete-user:
+                description: Delete a user in Aircall
```

Not sure I like it but feels more consistent with the rest
```suggestion
                description: Deletes a user in Aircall
```